msteams-platform/tabs/how-to/personal-tab-navigation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Personal tab app navigation
+description: Learn about the options for navigation within a personal tab app using the TeamsJS library.
+ms.date: 10/26/2022
+ms.topic: conceptual
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+
+# Navigate within a personal tab app
+
+The TeamsJS library provides support for navigation within a personal app. This article discusses the options available, which includes types of navigation such as that between tabs within the app or through use of Teams UI components like the back button.
+
+[!INCLUDE [sdk-include](~/includes/sdk-include.md)]
+
+Personal tabs provide a great way to enhance the Microsoft Teams experience. Using personal tabs, you can provide a user access to their application right within Teams, without the user having to leave the experience or sign in again. For more information about personal tabs and how you can extend them across Microsoft 365 products, see [Build tabs for Teams](~/tabs/what-are-tabs.md) and [Extend a Teams personal tab across Microsoft 365](~/m365-apps/extend-m365-teams-personal-tab.md).
+
+## Navigate between tabs
+
+The [pages](/javascript/api/@microsoft/teams-js/pages?view=msteams-client-js-latest&preserve-view=true) capability of the TeamsJS library provides support for navigation between tabs within a personal app. Specifically, the [pages.currentApp](/javascript/api/@microsoft/teams-js/pages.currentapp?view=msteams-client-js-latest&preserve-view=true) namespace offers a function `navigateTo(NavigateWithinAppParams)` to allow navigation to a specific tab within the current app and a function `navigateToDefaultPage()` to navigate to the first tab defined in the app's manifest.
+
+The following code illustrates how to navigate to a specific page:
+
+```js
+if (pages.currentApp.isSupported()) {
+    const navPromise = pages.currentApp.navigateTo({pageId: <pageId>, subPageId:<subPageId>});
+    navPromise.
+        then((result) => {/*Successful navigation*/}).
+        catch((error) => {/*Failed navigation*/});
+}
+else {/*Handle situation where capability isn't supported*/}
+```
+
+The following code illustrates how to navigate to the app's default tab:
+
+```js
+if (pages.currentApp.isSupported()) {
+    const navPromise = pages.currentApp.navigateToDefaultPage();
+    navPromise.
+        then((result) => {/*Successful navigation*/}).
+        catch((error) => {/*Failed navigation*/});
+}
+else {/*Handle situation where capability isn't supported*/}
+```
+
+It is also possible to use the `pages.navigateToApp()` function, `pages.tabs.navigateToTab()` function, or deep links for tab navigation, as shown in [Create deep links](~/concepts/build-and-test/deep-links.md#navigate-within-your-app), however the preferred method for apps that are extended across Microsoft 365 is to use the APIs in the `pages.currentApp` namespace.
```

Comment:
Would be good to avoid passive voice (per style guide). How about:

**You can also** use the `pages.navigateToApp()` function, `pages.tabs.navigateToTab()` function, or deep links for tab navigation,

Reply:
This has been changed as per style guide